lib/seahorn/BvOpSem2.cc

```diff
@@ -513,6 +515,44 @@ class OpSemVisitor : public InstVisitor<OpSemVisitor>, OpSemVisitorBase {
     llvm_unreachable(nullptr);
   }
 
+  void visitFatPointerInstr(CallSite CS) {
+    const Function *f = getCalledFunction(CS);
+    const Instruction &I = *CS.getInstruction();
+
```

Comment:
`assert(f);`

Reply:
still todo

Reply:
I don't recall the context here. But I assume we just want to make sure f is not null.

Reply:
Yes. `getCalledFunction` returns null if the call is indirect and cannot be resolved to a known function.